ballerina/tests/sql-department-tests.bal

```diff
@@ -1,6 +1,6 @@
-// Copyright (c) 2022 WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+// Copyright (c) 2023 WSO2 LLC. (http://www.wso2.org) All Rights Reserved.
 //
-// WSO2 Inc. licenses this file to you under the Apache License,
+// WSO3 Inc. licenses this file to you under the Apache License,
```

Comment:
```suggestion
// WSO2 Inc. licenses this file to you under the Apache License,
```

Reply:
Fixed in [945e4cb](https://github.com/ballerina-platform/module-ballerina-persist/pull/186/commits/945e4cbb91c854bcefc85fac4b7810c903a0125f)